tests/integration/framework-detection.test.js

```diff
@@ -69,7 +69,7 @@ describe.concurrent('frameworks/framework-detection', () => {
   // This test has a race condition that occasionally causes it to fail when run concurrently.
   // Running it in isolation (or removing the '.concurrent' on the describe block above)
   // fixes it. See CT-1094 for more details
-  test('should log the command if using static server and `command` is configured', async (t) => {
```

Comment:
See comment above ☝️ test is prevent us from releasing this so skipping it.

Reply:
Seems like there's a PR from @mrstork to address this -  https://github.com/netlify/cli/pull/6564/files. We can probably follow up there 👍